### PR TITLE
Introduces the CodeSee devtool

### PR DESCRIPTION
**Pull request by jleven**
@subho007, I believe we have CodeSee working with Irene!

Developers can make CodeSee recordings of interactions in the browser, explore the resulting data flows, and create and share notes linked to the code.

Helpful links:
- [The Quick Start Guide](https://docs.codesee.io/en/latest/quick-start/)
- CodeSee [App](app.codesee.io)
- CodeSee [homepage](codesee.io)

p.s. - I also made two tweaks to the README based on bumps in the road I ran into when getting the app running:
   - no need to run `bower install`
   - there appears to be a development requirement to have `watchman` installed.

### CODE REVIEW

**Comment by subho007** [comment]
Thanks @jleven for the PR, will get it reviewed and merged soon 👍

**Comment by subho007** [comment]
@jleven if you can ignore the eslint check for the require.resolve, the test would pass. 
The error generated currently is 

`Error:   13:21  error  "@codesee/babel-plugin-instrument" is extraneous  node/no-extraneous-require`

File: ember-cli-build.js [hunk 1]
```diff
@@ -4,10 +4,21 @@ var EmberApp = require('ember-cli/lib/broccoli/ember-app');
 var environment = EmberApp.env();
 var minifyEnabled = environment === "production" || environment === "staging";
 
+var babel = {
+  plugins: []
+};
+if (environment === 'development') {
+  // Add CodeSee instrumentation in development mode
+  babel.plugins.push([
+    require.resolve("@codesee/babel-plugin-instrument"), { frameworks: ["ember"]}
```
Comment (subho007) [inline ember-cli-build.js:13]: @jleven if you can ignore the eslint check for the require.resolve, the test would pass. 
The error generated currently is 

`Error:   13:21  error  "@codesee/babel-plugin-instrument" is extraneous  node/no-extraneous-require`

Comment (jleven) [reply ember-cli-build.js]: Thanks @subho007! It looks like we don't need `require.resolve` here at all, so I've just removed it. I also resolved the merge conflict and cleaned up the history while I was at it. Fingers crossed, this should be good to merge now 🤞

**Review by subho007 — APPROVED** [review]

**Comment by guganabu** [comment]
We have planned a feature release in near future and dev work is going on, so we will be merging the PR once the develop branch is ready for merge.
@jleven

**Pull request closed without merging**